backend/app/core/config.py

```
@@ -2,9 +2,11 @@
 アプリケーション設定
 """
 
-from typing import List, Any
-from pydantic_settings import BaseSettings
-from pydantic import Field, field_validator
+import json
+from typing import Any, Union
```

Flake8 (7.2.0)

[error] 'typing.Any' imported but unused

(F401)